### ISSUE TITLE
HUD Go to another level and screen do nothing.

### ISSUE THREAD

**Issue by extremestrategy** [TheGameCreators/GameGuruRepo#5130]
Inside of the GameGuru Max HUD system, there are two options that don't do anything, firstly the Go to another level, this should be able to send the player to another level, either through a node system, or a dropdown, but when selected neither option appears, the same holds true for Go to another screen, this option does nothing as well.

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/114613272/602cbbdc-d199-4ddd-8dd7-f00531df7ca1)

**Comment by LeeBamberTGC** [comment]
@extremestrategy Just tried the latest EXP build with he GO TO ANOTHER LEVEL and that seems to work as needed. See my storyboard layout with level 1 going to a SCREEN with a BUTTON set to "Go To Another Level" and it took me to the correct level that was specified in the storyboard (and I can move this to the other level and that works too):
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/c73a0160-7152-419e-acbb-fe1822349f47)

I then changed the button logic to Go to Another Screen, and then the node in the storyboard could not connect to a game level:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/3bcc47c4-2aec-4bfb-bd4b-a00a5c9b6341)

But could then connect to another screen:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/82dceb59-4432-489d-9d47-1a8235362d39)

This is the current expected behaviour, specify the logic for the button in the screen and then use the storyboard to connect the nodes up. Perhaps it would make sense to change the node graphics or color code them so it is more intuitive what can connect to what based on the level/screen node attributes?

**Comment by extremestrategy** [comment]
@LeeBamberTGC My apology for the misunderstanding, the problem is with a custom HUD screen, not a custom screen, custom screens work correctly, the problem is with a custom HUD, not a custom screen.

![image](https://github.com/TheGameCreators/GameGuruRepo/assets/114613272/baa75fd7-be18-48b2-9be1-914c6c583a20)

**Comment by LeeBamberTGC** [comment]
@extremestrategy So you are trying to open a HUD screen while inside a game level, then pressing a button in there to force the current level to end and to go to another screen/level?

**Comment by extremestrategy** [comment]
That is correct @LeeBamberTGC the option is there, however it doesn't do anything, create a HUD screen, add a button, go to action, and the options are there, they just don't do anything.

**Comment by LeeBamberTGC** [comment]
@extremestrategy I have changed this to a feature request and removed these actions from the HUD screen type to avoid user confusion.  There is another plan in play for regulating the logic of how levels are completed, and player death logic too.

**Comment by extremestrategy** [comment]
Works for me.
